javascript/javascript1/week1/main.js

```diff
@@ -2,20 +2,21 @@
 let yearOfBirth = 1989;
 let yearFuture = 2045;
 let age = yearFuture - yearOfBirth;
-console.log('Your age will be either ' + age + ' or ' + (age - 1));
+console.log('Your age will be' + age);
 
 /*2. Dog age calculator*/
 
 let dogYearOfBirth = 2019;
 let dogYearFuture = 2027;
 let humanYear = dogYearFuture - dogYearOfBirth;
-let shouldShowResultInDogYears = 7 * humanYear;
-let shouldShowResultInDogYears = true;
-if (shouldShowResultsInDogYears) {
-        console.log(`Your dog age will be ${shouldShowResultInDogYears} dog years old in 2027.`);
+let dogYear = 7 * humanYear;
+let shouldShowResultInDogYears = dogYear;
+
+if (shouldShowResultsInDogYears=true) {
```

Comment:
You want to check if the variable `shouldShowResultsInDogYear` is true. What you do is to assign `true` to variable `shouldShowResultsInDogYear` .  You can do something like this:
`let shouldShowResultInDogYears = true
if (shouldShowResultsInDogYears)` or `if (shouldShowResultsInDogYears === true)`